src/styles/index.scss

```
@@ -1,3 +1,46 @@
-body {
+@import 'header';
+@import 'card';
+@import 'stars';
+@import 'variables';
+
+* {
```

Don't use `*` selector for padding, margin, etc. just for `box-sizing`
make separate file for reset styles
```suggestion
* {
  box-sizing: border-box;
 }
```